python/cm/job.py

```diff
@@ -191,7 +191,7 @@ def check_action_config(
     if obj.config:
         cl = ConfigLog.objects.get(obj_ref=obj.config, id=obj.config.current)
         obj_conf = cl.config
-        current_attr = cl.attr
+        current_attr = cl.attr or None  # convert {} (empty dict) to None
```

Comment:
What for ?

Reply:
to prevent KeyError in cm.adcm_config.check_attr when comparing attr's kaeys and current_attr's (cluster attr == {}) keys